modules/virt-editing-vm-web.adoc

```diff
@@ -14,31 +14,40 @@
 ifeval::["{context}" == "virt-editing-vm-template"]
 :virt-vm-template:
 :object: virtual machine template
-:object-gui: Virtual Machines
-:object-vm-overview: VM Template Details
+:object-gui: Templates
+:object-vm-overview: Virtual Machine Overview
+:object-vm-details: VM Template Details
 endif::[]
 
 ifeval::["{context}" == "virt-edit-vms"]
 :virt-vm:
 :object: virtual machine
 :object-gui: Virtual Machines
 :object-vm-overview: Virtual Machine Overview
+:object-vm-details: Details
+:object-details: Details
 endif::[]
 
 [id="virt-editing-vm-web_{context}"]
 
 = Editing a {object} in the web console
 
-Edit select values of a {object} that displays in the *{object-vm-overview}* screen of the web console by clicking on the pencil icon next to the relevant field. Other values can be edited using the CLI.
+Edit select values of a {object} that displays in the *{object-vm-details}* screen of the web console by clicking on the pencil icon next to the relevant field. Other values can be edited using the CLI.
```

Comment:
Suggestion: "Edit select values of a {object} in the web console by clicking the pencil icon next to the relevant field."

Reply:
Change made.